lib/tasks/sync_with_inventory.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+desc 'Remove systems in Compliance DB which are not in the inventory'
+task sync_with_inventory: [:environment] do
+  ::Account.all.find_each do |account|
```

Comment:
Is this an N+1 case? I can't convince myself by testing that this is faster; what do you think?

```suggestion
  ::Account.includes(:hosts).find_each do |account|
```